src/lib/fauna.mjs

```diff
@@ -0,0 +1,31 @@
+export const getV10Client = async ({ url, secret }) => {
+  const { Client } = (await import("fauna")).default;
```

Comment:
also, i totally lied to you about there not being an existing v10 client usage in the project :(

the eval command has its own v10 query path that's used by it and the shell command:
https://github.com/fauna/fauna-shell/blob/v3/src/commands/eval.mjs#L101

i think it would make the most sense for eval and shell to only have the formatting logic, and for us to have only one implementation of the client/network stack. take a look at `getSimpleClient` and `performQuery` and see what you think?

Reply:
Yeah that v10 path is not using the driver though so that's why I set up these new helpers. I definitely agree about having a single implementation, I tried consolidating things a bit but it was proving to be difficult. Something to sort out next week :)